TravelerKit/Models/Airport.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
  Refer to Flight.swift for nested JSON structure
  **/
 
-public struct Airport: Decodable {
+public struct Airport: Decodable, Equatable {
```

Comment:
Doesn't this require an actual implementation for the `==` method?

Im surprised this even compiled

Reply:
Swift implements it by default: 
https://www.hackingwithswift.com/example-code/language/how-to-conform-to-the-equatable-protocol
https://developer.apple.com/documentation/swift/equatable

Reply:
Cool I didnt know this :)